components/resources/trezor_bridge_resources.grdp

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<grit-part>
+  <include name="IDR_TREZOR_BRIDGE_HTML" file="../trezor_bridge/resources/bridge.html" type="BINDATA" />
+  <include name="IDR_TREZOR_BRIDGE_IFRAME" file="../trezor_bridge/resources/trezor/iframe.html" type="BINDATA" />
+  <include name="IDR_TREZOR_BRIDGE_IFRAME_DATA_CONFIG" file="../trezor_bridge/resources/trezor/data/config.json" type="BINDATA" />
+  <include name="IDR_TREZOR_BRIDGE_IFRAME_DATA_COINS" file="../trezor_bridge/resources/trezor/data/coins.json" type="BINDATA" />
+  <include name="IDR_TREZOR_BRIDGE_IFRAME_DATA_MESSAGES_MESSAGES" file="../trezor_bridge/resources/trezor/data/messages/messages.json" type="BINDATA" />
```

Comment:
If you follow my advice on using GN `copy` to move the files here, then you would change these references to the generated directory

Reply:
done